tests/playbooks/tests_route_next_hop_ethernet.yml

```
@@ -0,0 +1,131 @@
+# SPDX-License-Identifier: BSD-3-Clause
+---
+- hosts: all
+
+- name: Test configuring ethernet devices
+  hosts: all
+  vars:
+    type: veth
+    interface0: ethtest0
+    interface1: ethtest1
+
+  tasks:
+    - name: "set type={{ type }} and interface={{ interface0 }}"
```

```suggestion
    - name: Set type and interface
```
might as well try to make the code ansible-lint clean, even if the other tests don't pass ansible-lint